rules/library/xcconfig.bzl

```diff
@@ -236,3 +236,93 @@ def settings_from_xcconfig(xcconfig):
         ibtool_copts = ibtool_copts,
         linkopts = linkopts,
     )
+
+def build_setting_name(build_setting):
+    """Returns the name of a given bazel build setting from the fully-qualified label
```

Comment:
why is this necessary? We should always be able to use a full label to refer to any target?

Reply:
This is used to name the files being written [here](https://github.com/bazel-ios/rules_ios/blob/40d41773b30e1b63fa6f87684da25fad54a0f9b3/rules/app.bzl#L64)

Without some kind of unique identifier per build setting rules with the duplicated names were being generated